lib/cmd/fh3/admin/domains/check.js

```diff
@@ -7,7 +7,6 @@ module.exports = {
     cmd : 'fhc admin domains check --domain=<domain>',
     desc : i18n._('Check the domain')}],
   'demand' : ['domain'],
-  'perm' : "cluster/reseller/customer:write",
```

Comment:
Why is this being removed?
Shouldn't we just check for the same permissions level as on the backend? ( https://github.com/fheng/fh-supercore/blob/4a41bcc0ede97525b4d8e6ca4dd5ca34f180737b/lib/permissions/permissions_map.js#L894 )